app/webpacker/components/Tickets/api/updateStatus.js

```diff
@@ -0,0 +1,19 @@
+import { fetchJsonOrError } from '../../../lib/requests/fetchWithAuthenticityToken';
+import { actionUrls } from '../../../lib/requests/routes.js.erb';
+
+export default async function updateStatus({ ticketId, status, currentStakeholder }) {
```

Comment:
This signature is a bit inconsistent about IDs. For the ticket, it expects the ID as a parameter directly, but for the stakeholder, it expects the full object and then extracts the ID by itself.
Is there any particular reason why you need to do it this way?